cbmgmtx/mgmt.go

```diff
@@ -81,6 +81,8 @@ func (h Management) DecodeCommonError(resp *http.Response) error {
 		// if we get this specific error, and its none of the above errors, then this indicates
 		// that it was the top level resource which could not be found, which is the bucket.
 		err = ErrBucketNotFound
+	} else if resp.StatusCode == 400 {
```

Comment:
Is this definitely correct? I can imagine that the server may respond with a 400 for more things than just invalid args?

Reply:
I forgot to update this, IIRC @brett19 and I agreed that `BadRequest` would be more appropriate, but correct me if I'm wrong